src/components/cylc/workflow/Lumino.vue

```diff
@@ -0,0 +1,166 @@
+<!--
+Copyright (C) NIWA & British Crown (Met Office) & Contributors.
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+-->
+<template>
+  <div>
+    <div id="workflow-panel">
+      <div ref="main" id="main" class="pa-4 fill-height"></div>
```

Comment:
I think this `id` won't necessarily be unique since it is in a template.

Reply:
Good point, I will confirm I don't have any CSS using `#main` (I think I had in the project where I wrote this component before moving here)

Reply:
Oh, it's actually used in the styles

![image](https://user-images.githubusercontent.com/304786/90494002-22e58580-e197-11ea-983e-9dc0b7f8c9c2.png)

Should we try to remove it here? Maybe use a class, or try `#workflow-panel { div:first-of-type { ... } }` ?

Reply:
Yeah, this is technically invalid HTML, a class should suffice?

Reply:
Done